### PR TITLE
Feature/header styles

### PR DESCRIPTION
**Pull request by KamiAmiga**

### CODE REVIEW

File: src/components/AddButton/AddButton.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <a @click=""></a>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'AddButton',
+        methods: {
+            addIcon(icon) {
+                console.log(icon);
```
Comment (jleveugle) [inline src/components/AddButton/AddButton.vue]: Outch :)

Remove `console.log` :)

File: src/components/AddButton/AddButton.vue [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            }
+        }
+    };
+</script>
+
+<style lang="scss" scoped>
```
Comment (jleveugle) [inline src/components/AddButton/AddButton.vue]: Not used :)

File: src/components/AppHeader/AppHeader.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<template>
+    <header class="header">
+      <h1 class="header__title">
+          <img class="header__title-logo" src="../../assets/logo.svg" alt="OVH group logo">
```
Comment (jleveugle) [inline src/components/AppHeader/AppHeader.vue]: `width` + `height`

File: src/components/AppHeader/LoginButton/LoginButton.vue [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<template>
+    <button class="loginButton">
```
Comment (jleveugle) [inline src/components/AppHeader/LoginButton/LoginButton.vue]: Don't forget the `type`

File: src/components/Homepage.vue [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+export default {
+    name: 'Homepage',
+    components: {
+        Icons
+    }
+}
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style lang="scss" scoped>
```
Comment (jleveugle) [inline src/components/Homepage.vue]: Not used

File: src/components/Icons.vue [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            add(icon) {
+                this.$store.state.moduleIcons.icons.push(icon);
+            }
+        },
+        computed: {
+            total() {
+                return this.$store.state.moduleIcons.icons.length;
+            },
+            icons() {
+                console.log(this.$store.state.moduleIcons.icons);
```
Comment (jleveugle) [inline src/components/Icons.vue]: -- `console.log`

File: src/styles/variables/_colors.scss [hunk 7]
```diff
@@ -0,0 +1,36 @@
+/* blues */
```
Comment (jleveugle) [inline src/styles/variables/_colors.scss]: UI Kit colors ?

File: src/components/AddButton/AddButton.vue (hunk 1 above)
Comment (frisouilles) [reply src/components/AddButton/AddButton.vue]: wip

File: src/components/AddButton/AddButton.vue (hunk 2 above)
Comment (frisouilles) [reply src/components/AddButton/AddButton.vue]: wip

File: src/components/Homepage.vue (hunk 5 above)
Comment (frisouilles) [reply src/components/Homepage.vue]: wip

File: src/components/Icons.vue (hunk 6 above)
Comment (frisouilles) [reply src/components/Icons.vue]: wip

**Comment by frisouilles** [comment]
It's a wip

**Pull request merged**